### PR TITLE
Added "Auto Assign" to automate PRs

### PR DESCRIPTION
**Pull request by BySwiizen**
### What is the purpose of this pull request?

1. Added Auto-Assign to automate PRs.

#### Changes

- [x] Use GitHub checklists to list things you've done or are still working on.

### CODE REVIEW

**Comment by BySwiizen** [comment]
In the meantime, I will add @datatags manually if necessary!

File: .github/auto-assign_config.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+addReviewers: true
+
+addAssignees: author
```
Comment (datatags) [inline .github/auto-assign_config.yml:3]:
```suggestion
```
It seems redundant to me to add the author as an assignee to their own PR

**Review by datatags — CHANGES_REQUESTED** [review]

File: .github/auto-assign_config.yml (hunk 1 above)
Comment (BySwiizen) [reply .github/auto-assign_config.yml:3]: This seems logical to me since it is he who is at the origin of the PR.

Comment (datatags) [reply .github/auto-assign_config.yml:3]: Ok 👍

**Review by datatags — APPROVED** [review]

**Pull request merged**